apps/sptribs/sptribs-cron-migrate-global-search-fields/sptribs-cron-trigger-complete-hearing-outcome.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: sptribs-cron-migrate-global-search-fields
+spec:
+  releaseName: sptribs-cron-migrate-global-search-fields
```

Comment:
(May or may not be related to the issue you are facing with the build)
The file name is wrong, it should be related global-search.

Reply:
Hmmm. The fix didn't resolve the build issue, should be something else then.